test/unit/sessions/softphone-session-handler.test.ts

```diff
@@ -233,7 +233,7 @@ describe('acceptSession()', () => {
     const startMediaSpy = jest.spyOn(mockSdk.media, 'startMedia');
 
     const element = {};
-    const volume = mockSdk._config.defaults.audioVolume = 67;
+    const volume = mockSdk._config.defaults!.audioVolume = 67;
```

Comment:
Codacy has a fix for the issue: [Forbidden non-null assertion.](https://app.codacy.com/gh/MyPureCloud/genesys-cloud-webrtc-sdk/pullRequest?prid=9630720)
```suggestion
    const volume = mockSdk._config.defaults?.audioVolume = 67;
```